### ISSUE TITLE
Support Symfony/Process v5

### ISSUE THREAD

**Issue by ob-ivan** [ob-ivan/diversitest#24]
Since Symfony/Process v5, `Symfony\Component\Process\Process::__construct()` changed from accepting `string|array` to just `array`.

~~This means that `Ob_Ivan\DiversiTest\PackageManager::getCommands()` has to return an array of `string[]`.~~

This means that `\Ob_Ivan\DiversiTest\DiversiTestCommand::runCommand()` has to call `Process::fromShellCommandline()` once it is supported.

**Comment by ob-ivan** [comment]
At the same time, [the documentation of Symfony/Process 3.0](https://symfony.com/doc/3.0/components/process.html) mentions the usage as:

```php
$process = new Process('ls -lsa');
```

And the constructor's signature does not support arrays:

```php
    /**
     * Constructor.
     *
     * @param string         $commandline The command line to run
```

**Comment by ob-ivan** [comment]
If `\Ob_Ivan\DiversiTest\PackageManager::$commandLine` remains a string, then it's difficult to use it to build a command as an array of words.

Actually, it is not a command line, it is a **command line template**.

So what I need to come up with is a way to configure a template for generating commands an arrays of words.

- [x] But before that, I would like to replace a static factory `\Ob_Ivan\DiversiTest\PackageManager::fromConfig()` with a separate factory class.

**Comment by ob-ivan** [comment]
Solved via f6b9e1f1e09ff30a5f3a4c036a7992a4fa842bab.
